docs/source/examples/plots_examples.rst

```diff
@@ -1,6 +1,182 @@
 Plotting with manim
 =================================
 
+Examples to illustrate the use of GraphScenes in manim
+
+
+.. manim:: Plot1
+    :quality: medium
+    :save_last_frame:
+
+    class Plot1(GraphScene):
+        def construct(self):
+            self.setup_axes()
+            func_graph=self.get_graph(lambda x: np.sin(x))
+            self.add(func_graph)
+
+.. manim:: Plot2yLabel
+    :quality: medium
+    :save_last_frame:
+
+    class Plot2yLabel(GraphScene):
+        CONFIG = {
+            "y_min": 0,
+            "y_max": 100,
+            "y_tick_frequency": 10,
+            "y_labeled_nums": np.arange(0, 100, 10)
+        }
+
+        def construct(self):
+            self.setup_axes(animate=True)
+            dot = Dot().move_to(self.coords_to_point(PI / 2, 20))
+            func_graph = self.get_graph(lambda x: 20 * np.sin(x))
+            self.add(func_graph)
+            self.add(dot)
+
+.. manim:: Plot3DataPoints
+    :quality: medium
+    :save_last_frame:
+
+    class Plot3DataPoints(GraphScene):
+        CONFIG = {
+            "y_axis_label": r"Concentration[$\%$]",
+            "x_axis_label": "Time [s]",
+            }
+
+        def construct(self):
+            data = [1, 2, 2, 4, 4, 1, 3]
+            self.setup_axes(animate=True)
+            for time, dat in enumerate(data):
+                dot = Dot().move_to(self.coords_to_point(time, dat))
+                self.add(dot)
+
+.. manim:: Plot3bGaussian
+    :quality: medium
+    :save_last_frame:
+
+    amp = 5
+    mu = 3
+    sig = 1
+
+    def gaussian(x):
+        return amp * np.exp((-1 / 2 * ((x - mu) / sig) ** 2))
+
+    class Plot3bGaussian(GraphScene):
+        def construct(self):
+            self.setup_axes()
+            graph = self.get_graph(gaussian, x_min=-1, x_max=10).set_stroke(width=5)
+            self.add(graph)
+
+.. manim:: Plot3cGaussian
+    :quality: medium
+    :save_last_frame:
+
+    class Plot3cGaussian(GraphScene):
+        def construct(self):
+            def gaussian(x):
+                amp = 5
+                mu = 3
+                sig = 1
+                return amp * np.exp((-1 / 2 * ((x - mu) / sig) ** 2))
```

Comment:
`Plot3bGaussian` and `Plot3cGaussian` should use the same strategy w.r.t. `gaussian` and `amp, mu, sig`.

Reply:
In this case, b and c have the purpose of showing that is possible on the one hand to define math functions outside, and on the other hand inside the construct class.
One is the more pythonic way, the other way is the tighter and more convenient way in my opinion.
Here, it just illustrates that both is possible.
In a future PR, I can write some things to the purpose of these examples.

Reply:
I prefer to have `gaussian` outside `construct` (and outside `Scene`, really) bc it seems more pythonic. But I agree it should be illustrated that both are possible. Not sure if two videos/images are necessary for this tho, maybe just include one and mention that the alternative is possible?

Reply:
Most times as a reader, I would prefer two explicit examples over one example and the sentence "and you can also do this"

Reply:
I think it's fine to do that, especially if we're trying to encourage one and not the other. If one is more pythonic, I think we should encourage that one.

"There should be one-- and preferably only one --obvious way to do it."

Reply:
> Even if the code looks not perfect, what matters is the resulting video and not that it is coded in the way that is to a most common convention.

I don't care for perfect code, I care for consistency. Whatever the consensus is, we should stick to it as much as possible.

Reply:
For me, manim has a very strong advantage that you can do things in lots of different ways, depending on your current situation.
With this example, I want to showcase that we have here two convenient ways of doing things.
I would be fine with adding one line like:
`Here is a script that shows how to define a function in a most pythonic way`
and to the other one:
`here is a script that gives you the opportunity to define your function close to where it's needed, which might be useful in the case of long scripts.`

Reply:
I think part of my hesitation here is that these two examples are really illustrating a python feature, not a manim feature. But I'll stop now and let you and others decide :)

Reply:
In principle, I agree with @leotrs: we should primarily showcase manim features over python features.

Anyhow, I think we can still merge this as-is. I am pretty sure that we will revisit these examples sooner or later to add some more explanations and stuff -- and then we might even have a better idea of what precisely should be illustrated with these examples.

Reply:
Agreed, let's merge as is and continue iterating rapidly :)